src/com/company/exercises/MaxMin.java

```diff
@@ -0,0 +1,48 @@
+package com.company.exercises;
+
+
+
+import com.company.data.MusicSampleData;
+import com.company.models.Album;
+import com.company.models.Track;
+
+import java.util.Comparator;
+import java.util.LinkedList;
+import java.util.List;
+
+import static java.util.Arrays.asList;
+
+public class MaxMin {
+    public static void main(String[] args) {
+        List<Track> tracks = asList(new Track("Bakai", 524),
+                new Track("Violets for Your Furs", 378),
+                new Track("Time Was", 451));
+
+        Track shortestTrack = tracks.stream()
+                .min(Comparator.comparing(track -> track.getLength()))
+                .get();
+
+        System.out.println(shortestTrack.getName());
+
+        List<Album> albums = new LinkedList<>();
+        albums.add(MusicSampleData.aLoveSupreme);
+        albums.add(MusicSampleData.sampleShortAlbum);
+        albums.add(MusicSampleData.manyTrackAlbum);
+
+
+
+        Album albumResult = albums.stream()
```

Comment:
Names should generally try to communicate why the object exists, what is its purpose. I would name this Album something like albumWithLeastTracks. Its not perfect, maybe we can come up with something better if we spend the time, but it is better than albumResult in my opinion.

Reply:
Done.